src/doc/man/cargo-vendor.md

```diff
@@ -21,6 +21,11 @@ stdout after `cargo vendor` completes the vendoring process.
 You will need to add or redirect it to your Cargo configuration file,
 which is usually `.cargo/config.toml` locally for the current package.
 
+By default, Cargo treats vendored crates as read-only as it does crates.io.
```

Comment:
“By default” sounds like there is a way to tweak it, though there is no way to change the read-only property.

```suggestion
Cargo treats vendored sources as read-only as it does to registry and git sources.
```